scss/input/_layout.scss

```diff
@@ -13,6 +13,7 @@
     $checkbox-dash-offset-x: $checkbox-border-width * 2 !default;
     $checkbox-dash-offset-y: ( $checkbox-size - $checkbox-dash-height ) / 2 !default;
 
+    $textbox-default-font-size: $font-size !default;
```

Comment:
I'd remove the `-default` from the name. Indeed, there's `$textbox-default-width`, but that name isn't perfect either.